### PR TITLE
fix: scripts

### PR DESCRIPTION
**Pull request by lemaitre-aneo**

### CODE REVIEW

File: infrastructure/utils/scripts/deploy-dev-test-infra.sh [hunk 1]
```diff
@@ -1,18 +1,18 @@
 #! /bin/bash
```
Comment (aneojgurhem) [inline infrastructure/utils/scripts/deploy-dev-test-infra.sh]:
```suggestion
#! /usr/bin/env bash
```

File: infrastructure/utils/scripts/installation/prerequisites/install-docker.sh [hunk 2]
```diff
@@ -1,15 +1,6 @@
-#!/bin/bash
+#! /bin/sh
```
Comment (aneojgurhem) [inline infrastructure/utils/scripts/installation/prerequisites/install-docker.sh:1]: env too ?

File: infrastructure/utils/scripts/get_node_ip.sh [hunk 3]
```diff
@@ -1,8 +1,8 @@
-#! /bin/bash
+#! /bin/ sh
```
Comment (aneojgurhem) [inline infrastructure/utils/scripts/get_node_ip.sh:1]:
```suggestion
#! /bin/sh
```

File: infrastructure/utils/scripts/installation/prerequisites/install-docker.sh [hunk 4]
```diff
@@ -1,15 +1,6 @@
...
 # This script is used to install docker
 
 # https://docs.docker.com/engine/install/ubuntu/
 sudo apt update
```
Comment (aneojgurhem) [inline infrastructure/utils/scripts/installation/prerequisites/install-docker.sh:5]: you kept the update and the comments above

File: tools/number_processed_tasks_per_pod.sh [hunk 5]
```diff
@@ -1,9 +1,11 @@
...
 kubectl get secret -n armonik mongodb-user-certificates -o jsonpath="{.data.chain\.pem}" | base64 --decode > ./mongodb_chain.pem
-MONGO_IP=$(kubectl get svc --selector="service=mongodb" -n armonik -o custom-columns="IP:.spec.clusterIP" --no-headers=true)
-docker run -it -v $(pwd)/mongodb_chain.pem:/chain.pem --rm rtsp/mongosh mongosh --tlsCAFile /chain.pem --tlsAllowInvalidCertificates --tlsAllowInvalidHostnames --tls -u $MUSER -p $MPASS mongodb://$MONGO_IP:27017/database #--eval 'db.serverStatus()'
+MONGO_IP="$(kubectl get svc --selector="service=mongodb" -n armonik -o custom-columns="IP:.spec.clusterIP" --no-headers=true)"
+docker run -it -v "$(pwd)/mongodb_chain.pem:/chain.pem" --rm rtsp/mongosh mongosh --tlsCAFile /chain.pem --tlsAllowInvalidCertificates --tlsAllowInvalidHostnames --tls -u "$MUSER" -p "$MPASS" "mongodb://$MONGO_IP:27017/database" #--eval 'db.serverStatus()'
 
 # Get the number of processed tasks by pod
-db.TaskData.aggregate([{$group : {_id:"$OwnerPodId", count:{$sum:1}}}, {$sort: {count:1}}]).forEach(printjson);
\ No newline at end of file
+echo 'db.TaskData.aggregate([{$group : {_id:"$OwnerPodId", count:{$sum:1}}}, {$sort: {count:1}}]).forEach(printjson);'
```
Comment (aneojgurhem) [inline tools/number_processed_tasks_per_pod.sh:11]: that will be shown after connection to the container

File: infrastructure/utils/scripts/installation/prerequisites/install-docker.sh (hunk 2 above)
Comment (lemaitre-aneo) [reply infrastructure/utils/scripts/installation/prerequisites/install-docker.sh:1]: `sh` is guaranteed to be located at `/bin/sh`, `bash` is not

**Review by aneojgurhem — APPROVED** [review]

**Pull request merged**